spec/support/common_client_stubs.rb

```diff
@@ -0,0 +1,43 @@
+shared_context :stub_api_following_standards do
+  let(:response) do
+    {
+        'data' => [ data ].flatten,
+        'page' => 1,
+        'page_size' => 1,
+        'total' => 1
```

Comment:
What about having `data` be an array, and then this could be:
``` ruby
{
  'data' => data,
  'page' => 1,
  'page_size' => data.length,
  'total' => data.length
}
```